prime.py

```diff
@@ -0,0 +1,18 @@
+def is_prime(num):
+    if num <= 1:
+        return False
+    for i in range(2, int(num**0.5) + 1):
+        if num % i == 0:
+            return False
+    return True
+
+def print_primes(n):
```

Comment:
Change the name  @here
```suggestion
def print_prime(n):
```

Reply:
Got it